packages/components/src/tools-panel/types.ts

```diff
@@ -12,6 +12,11 @@ export type ToolsPanelProps = {
 	 * The child elements.
 	 */
 	children: ReactNode;
+	/**
+	 * Flags that the items in this ToolsPanel will be contained within an inner
+	 * wrapper element allowing the panel to lay them out accordingly.
```

Comment:
I'd be happy to add something to that effect here. 

The current omission of any mention of the SlotFill pattern is a result of two separate pieces of feedback received on other PRs. The first was requesting the prop's description in the [readme be disconnected from the SlotFill](https://github.com/WordPress/gutenberg/pull/34632#discussion_r704339029) pattern. The second was for the [descriptions here in `types.ts` to match the README](https://github.com/WordPress/gutenberg/pull/34028#discussion_r712283742) in the ideal case. 

If we are happy with that slight divergence, I'll make the change. I'm easy either way 🙂

Reply:
Mhh, I see. I wonder if we can find a middle ground and somehow include info about how to use `ToolsPanel` with `SlotFill` — maybe the description could be slightly vaguer? Not a big deal anyway, I'm also good to keep it as-is

Reply:
Perhaps we could add that documentation around how to use the `ToolsPanel` with `SlotFill` closer to where that happens?

It could possibly go in the [InspectorControls README](https://github.com/WordPress/gutenberg/blob/9385d9c0c4a7a373e2c757d06bd71f8d989d6167/packages/block-editor/src/components/inspector-controls/README.md) where the [`BlockSupportToolsPanel`](https://github.com/WordPress/gutenberg/blob/e5809a887ef94fa9b90f896d6f3436fa3e325da7/packages/block-editor/src/components/inspector-controls/block-support-tools-panel.js) exists and illustrates the injection of items into the panel.

Reply:
That sounds like a good compromise :)

Reply:
When making the previous suggestion I was forgetting that the grouped inspector controls are still experimental and we probably shouldn't be documenting them at this stage. 

Would you be happy enough if we held off adding this documentation until that stabilizes?